config/routes.rb

```diff
@@ -14,7 +14,7 @@
   root 'landing_page#index'
   concern :searchable, Blacklight::Routes::Searchable.new
 
-  resource :catalog, only: [:index], as: 'catalog', path: '/catalog', controller: 'catalog' do
+  resource :catalog, only: [], as: 'catalog', path: '/catalog', controller: 'catalog' do
```

Comment:
This is the only thing I had to update manually. Apparently only doesn't take :index anymore. This change is reflected in blacklight 8/9 as well https://github.com/projectblacklight/blacklight/commit/74008cfc23f44f7857299ab1f90b4e69064f2ccd